src/uu/sort/Cargo.toml

```
@@ -25,6 +25,7 @@ memchr = { workspace = true }
 rand = { workspace = true }
 rayon = { workspace = true }
 self_cell = { workspace = true }
+sysinfo = {workspace = true}
```

It makes no difference at all but:
```suggestion
sysinfo = { workspace = true }
```